README.md

```diff
@@ -146,7 +146,11 @@ Styles are not added on `import/require()`, but instead on call to `use`/`ref`.
 ### `hmr`
 
 Enable/disable Hot Module Replacement (HMR), if disabled no HMR Code will be added.
-This could be used for non local development and production.
+If no specific value provided, the default value will be determined by the npm script that invoked webpack:
+  * If invoked via `npm preblish/prepublishOnly`, considered for production distribution, default to false.

```

Comment:
Typo here -- "preblish" should be "prepublish". Also "considered for production distribution" is grammatically strange. Consider just removing that phrase.